### PR TITLE
Domain stats

### PR DESCRIPTION
**Pull request by andrewxhill**
okay, simply tracks the number of unique trackers that have been seen on each top level domain and makes those available. good enough to move forward with for now

### CODE REVIEW

File: test/tests/domains.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+(function() {
+  QUnit.module("Domains");
+
+    //  todo... classic
```
Comment (sanderpick) [inline test/tests/domains.js:4]: haha

**Comment by sanderpick** [comment]
sweet!

**Review by sanderpick — APPROVED** [review]
very nice dude

File: test/tests/domains.js (hunk 1 above)
Comment (andrewxhill) [reply test/tests/domains.js:4]: haha

**Pull request merged**